addons/transifex/models/ir_translation.py

```diff
@@ -8,17 +8,58 @@
     from ConfigParser import ConfigParser
 from os.path import join as opj
 import os
+import re
 import werkzeug
 
-from odoo import models, fields
+from odoo import models, fields, _
 from odoo.modules.module import ad_paths
 
+REGEX_PLACEHOLDERS = re.compile("%(\(\w+\)|)([0+-]|)(\d+|\*|)[diouxXcrsa]|%(\(\w+\)|)([0+-]|)(\d+|\*|)(\.(\d+|\*|)|)[eEfFgG]|%%")
+REGEX_XML_TAGS = re.compile("</?(\w+)")
 
 class IrTranslation(models.Model):
 
     _inherit = 'ir.translation'
 
     transifex_url = fields.Char("Transifex URL", compute='_get_transifex_url')
+    is_valid = fields.Boolean(default=True)
+
+    def _compute_validity(self):
```

Comment:
We often use `_compute_XXX` for compute= callbacks, might be good to use a different name.

Reply:
actually, couldn't the `is_valid` field be a non-stored computed field? :thinking:

Reply:
Don't see why not, I assumed you had a reason to make it something else.

Reply:
Not really, this is a sous-marin task without clear spec and the purpose is "make translations with transifex better"

Reply:
OK then yeah it'd make sense to make it a computed field, and there doesn't seem to be much reason to store it (it's only used on the tnx list view, probably).

Reply:
Thinking about it there's a reason to store it: being able to filter on "bad" translations.